TRIAGING.md

```diff
@@ -1,8 +1,98 @@
+1. Duplicate?
+    * If you've seen this issue before [close it][], and go to the last step.
+    * Check if there are comments that link to a dupe. If so verify that this is indeed a dupe, [close it][], and go to the last step.
+1. Bugs:
+    * Label `Type: Bug`
+    * Reproducible? - Steps to reproduce the bug are clear. If they are not, ask for a clarification. If there's no reply after a week, [close it][].
+    * Reproducible on master?
+
+1. Non bugs:
+    * Label `Type: Feature`, `Type: Chore`, or `Type: Perf`
+    * Belongs in core? – Often new features should be implemented as a plugin rather than an addition to the core.
+      If this doesn't belong, [close it][], and go to the last step.
+    * Label `needs: breaking change` - if needed
+    * Label `needs: public api` - if the issue requires introduction of a new public API
+1. Label `frequency: *` – How often does this issue come up? How many developers does this affect?
+    * low - obscure issue affecting a handful of developers
+    * moderate - impacts a common usage pattern
+    * high - impacts most or all Hoodie apps
+1. Label `severity: *` - How bad is the issue?
+    * regression
+    * memory leak
+    * broken expected use - it's hard or impossible for a developer using Hoodie to accomplish something that Hoodie should be able to do
+    * confusing - unexpected or inconsistent behavior; hard-to-debug
+    * inconvenience - causes ugly/boilerplate code in apps
+1. Label `starter` - These issues are good targets for PRs from the open source community. Apply to issues where the problem and solution are well defined in the comments, and it's not too complex.
+
+1. Label `milestone: *` – Assign a milestone:
+   * Backlog - triaged fixes and features, should be the default choice
+   * x.y.z - e.g. 0.3.0
+
+
+1. Unassign yourself from the issue
+
+## Closing an Issue or PR
+
+We're grateful to anyone who takes the time to submit an issue, even if we ultimately decide not to act on it.
+Be kind and respectful as you close issues. Be sure to follow the [code of conduct][].
+
+1. Always thank the person who submitted it.
+1. If it's a duplicate, link to the older or more descriptive issue that supersedes the one you are closing.
+1. Let them know if there's some way for them to follow-up.
+    * When the issue is unclear or reproducible, note that you'll reopen it if they can clarify or provide a better example. Mention [jsbin](https://jsbin.com) for examples. Watch your notifications and follow-up if they do provide clarification. :)
+    * If appropriate, suggest implementing a feature as a third-party module.
+
+If in doubt, ask a core team member what to do.
+
+**Example:**
+
+> Thanks for submitting this issue!
+> Unfortunately, we don't think this functionality belongs in core.
+> The good news is that you could easily implement this as a plugin and publish it to npm with the `hoodie-plugin` keyword.
```

Comment:
I know this is example text but:

Do we have an example of how to implement a `hoodie-plugin` yet? If not, it might be difficult to tell someone they can "easily" implement one without context or reference, or even documentation on how to get started with building a plugin.